plugin/src/main/kotlin/com/ritense/plugin/autodeployment/PluginAutoDeploymentEventListener.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ *  Licensed under EUPL, Version 1.2 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.ritense.plugin.autodeployment
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.fasterxml.jackson.module.kotlin.treeToValue
+import com.ritense.plugin.domain.PluginConfiguration
+import com.ritense.plugin.service.PluginService
+import mu.KLogger
+import mu.KotlinLogging
+import org.springframework.boot.context.event.ApplicationReadyEvent
+import org.springframework.context.event.EventListener
+import org.springframework.core.io.Resource
+import org.springframework.core.io.ResourceLoader
+import org.springframework.core.io.support.ResourcePatternUtils
+import org.springframework.transaction.annotation.Transactional
+import java.io.IOException
+import java.lang.Exception
+
+@Transactional
+class PluginAutoDeploymentEventListener(
+    private val resourceLoader: ResourceLoader,
+    private val pluginService: PluginService,
+    private val objectMapper: ObjectMapper
+) {
+
+    @Transactional
+    @EventListener(ApplicationReadyEvent::class)
```

Comment:
Should include an order here to ensure plugins get autodeployed before process links are.

Reply:
process links deployment already has @Order(Ordered.LOWEST_PRECEDENCE) ensuring process links are the last to run